pkg/operator/etcd/pod.go

```diff
@@ -315,3 +327,20 @@ func (spec *Spec) Args() []string {
 
 	return flags.FormatArgs()
 }
+
+func annotationHash(m map[string]string) string {
```

Comment:
This should go in a new package like `pkg/operator/stringmaps` since we need it in multiple places already. We can then call the function `HashKeys()` to make clear that we're only hashing the keys.

Reply:
Done.